docs/getting-started/orchestration_quickstart.md

```diff
@@ -0,0 +1,162 @@
+14:36:55.567 | INFO    | prefect.engine - Created flow run 'dazzling-hawk' for flow 'Repo Info'
+14:36:55.570 | INFO    | Flow run 'dazzling-hawk' - View at https://app.prefect.cloud/account/0ff44498-d380-4d7b-bd68-9b52da03823f/workspace/aaf96bd2-298c-4fe4-98b0-6b4d520951e1/flow-runs/flow-run/5a891994-8ac3-44e5-894a-1b2f801c9333
+14:36:56.497 | INFO    | Flow run 'dazzling-hawk' - Created task run 'get_contributors-0' for task 'get_contributors'
+14:36:56.499 | INFO    | Flow run 'dazzling-hawk' - Executing 'get_contributors-0' immediately...
+14:36:57.405 | INFO    | Task run 'get_contributors-0' - Finished in state Completed()
+14:36:57.525 | INFO    | Flow run 'dazzling-hawk' - Created task run 'calculate_average_commits-0' for task 'calculate_average_commits'
+14:36:57.526 | INFO    | Flow run 'dazzling-hawk' - Executing 'calculate_average_commits-0' immediately...
+14:36:58.227 | INFO    | Task run 'calculate_average_commits-0' - Finished in state Completed()
+14:36:58.256 | INFO    | Flow run 'dazzling-hawk' - PrefectHQ/prefect repository statistics 🤓:
+14:36:58.257 | INFO    | Flow run 'dazzling-hawk' - Stars 🌠 : 12160
+14:36:58.258 | INFO    | Flow run 'dazzling-hawk' - Forks 🍴 : 1252
+14:36:58.259 | INFO    | Flow run 'dazzling-hawk' - Average commits per contributor 💌 : 345.07
+14:36:58.494 | INFO    | Flow run 'dazzling-hawk' - Finished in state Completed('All states completed.')
+```
+</div>
+
+
+Beyond examining these logs, you have the option to explore the FlowRun via the UI to visualize its dependency diagram. You should find a link directing you to the FlowRun page conveniently positioned at the top of your flow logs.
+
+Local execution is great for development and testing, but in order to schedule flow runs or trigger them based on events, you’ll need to [deploy](/tutorial/deployments/) your flows.
+
+
+### Step 5: Deploy Flow
+
+Deploying your flows is, in essence, the act of informing the Prefect API of where, how, and when to run your flows. Prefect offers CLI commands for quick deployment creation.
+
+!!! warning "Run `prefect deploy` commands from the **root** of your repo!"
+    When running any `prefect deploy` or `prefect init` commands, double check that you are at the root of your repo, otherwise the worker may attempt to use an incorrect flow entrypoint during remote execution!
+
+When you execute the deploy command, Prefect will automatically detect any flows defined in your repository. Simply choose the one you wish to deploy. Then, follow the 🧙 wizard to name your deployment, add an optional schedule, create a Work Pool, optionally configure a flow code pull step, and more!
+
+```bash
+prefect deploy
+```
+The last prompt in the `prefect deploy` wizard asks if you would like to save the configuration for the deployment, saying yes to this will result in a prefect.yaml file populated with your first deployment. You can use this yaml file to edit and [manage all deployments](/concepts/deployments-ux/) for this repo.
```

Comment:
```suggestion
The last prompt in the `prefect deploy` wizard asks if you would like to save the configuration for the deployment. Confirming this will result in a `prefect.yaml` file populated with your first deployment. You can use this yaml file to edit and [manage all deployments](/concepts/deployments-ux/) for this repo.
```